pkg/mesh/app.go

```diff
@@ -36,67 +32,6 @@ const (
 	listenerCheckRegex    = "^listener_manager.workers_started"
 )
 
-// StartApp uses the reminder of the command line to exec an app, using K8S_UID as UID, if present.
-func (kr *KRun) StartApp() {
-	var cmd *exec.Cmd
-	if len(os.Args) == 1 {
```

Comment:
I'm not sure why you had to make the changes - simply not passing any args to the container should have worked ? The code was written to already account for this case ( no app exec )

Reply:
ack, didn't know we still wanted to support single container usecases.

Reply:
Done, no app exec will still execute kr.Signals which is needed in multi-container too.